src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ImportStatusBauServiceTests.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Model;
+using Microsoft.Azure.Cosmos.Table;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.TableStorageTableNames;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class ImportStatusBauServiceTests
+    {
+        private static ImportStatusBauService BuildImportStatusBauService(List<DatafileImport> queryResults)
```

Comment:
Would put private methods below where they're being used in the file. Public methods should have the most prominance as they're the most impotant.

For tests, this usually means being at the bottom.